config/settings/base.py

```diff
@@ -33,6 +33,14 @@
 DATABASES["default"]["ATOMIC_REQUESTS"] = True
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+SECONDARY_DB_ALIAS = "default"
+
+if env.db("SECONDARY_DATABASE_URL", default=""):
```

Comment:
should this look up `env` rather than `env.db`?

Reply:
env.db is just a util to read the flat URL style spec for DB, right?

Reply:
Yeah, it transforms the url into a database dict, but there is no reason to do that operation since we are throwing away the result? It isn't particularly expensive but doesn't buy us anything and throws warnings when it hits the default value. We just care if the key exists, which `env` does without `.db`

Reply:
It's easier to define it in this way, with less num of vars to keep track of (like PASSWORD, DB, USERNAME etc). And the current primary DB is also setup that way (so is inconsistent). Do you have strong feelings against this?

Reply:
I'm not sure I understand, its exactly the same number of vars, I'm just asking for this line to remove `.db`. Definitely use `.db` when we setup the actual dict lower down, just not for the existence check (since that isn't what it is meant for, and is what the plain `env` call does).